src/highlight/HighlightCanvas.tsx

```diff
@@ -1,14 +1,24 @@
 import * as React from 'react';
 import { bdlYellorange, black, white } from 'box-ui-elements/es/styles/variables';
-import { AnnotationHighlight, Rect } from '../@types';
+import { Rect } from '../@types';
 import './HighlightCanvas.scss';
 
+type CanvasShapeBase = {
+    id?: string;
+};
+
+export type CanvasShapeRect = CanvasShapeBase & Rect;
+
+export type CanvasShape = CanvasShapeRect;
```

Comment:
Are these separate aliases needed or can we simplify this to

```
export type CanvasShape = Rect & { id?: string };

Reply:
We can simplify that for now, I was just trying to lay the baseline for extending to a drawing shape too

Reply:
I don't expect drawings would use the HighlightCanvas component, though. Probably best to simplify the code if/until we need the complexity.